lighthouse-core/audits/byte-efficiency/unused-css-rules.js

```diff
@@ -19,7 +19,9 @@ const UIStrings = {
 
 const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
 
-const IGNORE_THRESHOLD_IN_BYTES = 2048;
+// Allow 10KB of unused CSS to permit `:hover` and other styles not used on a non-interactive load.
+// @see https://github.com/GoogleChrome/lighthouse/issues/9353 for more discussion.
```

Comment:
I wish we had a better link for this discussion (there's a lot of other stuff going on in that thread :)